pkgs/development/libraries/libdigidocpp/default.nix

```diff
@@ -10,6 +10,17 @@ stdenv.mkDerivation rec {
      sha256 = "sha256-U5i5IAyJF4359q6M6mQemEuG7+inPYIXqLy8GHv4dkg=";
   };
 
+  patches = [
```

Comment:
I can confirm this fixes signing (and validating signature) - related crashes in qdigidoc4.

Let's ask upstream if they plan releasing this though, and link to the issue from here.

Reply:
I opened https://github.com/open-eid/libdigidocpp/issues/474. Can you add a comment to this issue?

Reply:
Isn't it obvious that the commit is cherry-picked from `master` and must thus appear in the next release?

Your issue might get us the next release a little sooner, but I fail to see the value it provides as link/comment in nixpkgs.

Reply:
Other maintainers are aware upstream has already been pinged, and can easily get there, without searching through the upstream issue tracker, or downstream PRs.

Reply:
Link added.